src/components/Card/Card.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <article class="location-card">
+  <article class="location-card" :id="id">
+    <div class="button-container">
+      <button class="button button-refresh" type="button" name="button" v-on:click.prevent="refreshLocation">🔄</button>
```

Comment:
Thinking we should bring in actual icons for these rather than text/emojis during the design phase of our project.